### ISSUE TITLE
github actions: brew audit is broken

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/homebrew-tap#27]
Seems like it doesn't take a path or filename any more.

**Comment by m3nu** [comment]
Yeah, just noticed too.

**Comment by ThomasWaldmann** [comment]
@m3nu do you know how to fix this easily?

I did some web search, but didn't find a solution.

Just noticed that `brew test-bot` seems to be popular within such actions.

**Comment by m3nu** [comment]
I'll see what the canonical way is right now.

**Comment by ThomasWaldmann** [comment]
Still broken:
```
Run brew audit --strict borgbackup-fuse.rb
Warning: audit is a developer command, so Homebrew's
developer mode has been automatically turned on.
To turn developer mode off, run:
  brew developer off

Error: Calling brew audit [path ...] is disabled! Use brew audit [name ...] instead.
Error: Process completed with exit code 1.
```

**Comment by ThomasWaldmann** [comment]
Fixed by #39, thanks @fasmat!
